build.gradle.kts

```diff
@@ -32,7 +32,7 @@ allprojects {
     configurations.all {
         resolutionStrategy {
             failOnDynamicVersions()
-            failOnChangingVersions()
+//            failOnChangingVersions()
```

Comment:
nit: do we need this or can be deleted?

Reply:
ideally we keep this, not sure if we can yet